admin-dev/themes/default/scss/rtl.scss

```diff
@@ -1,21 +1,379 @@
+            }
+          }
+        }
+
+        .timeline {
+          .timeline-item {
+            .timeline-caption {
+              .timeline-panel {
+                .panel-heading {
+                  a {
+                    &.btn {
+                      font-family: Vazirmatn, sans-serif;
+                    }
+                  }
+                  font-family: Vazirmatn, sans-serif;
+                }
+              }
+            }
+          }
+        }
+
+        .btn-group-action {
+          .btn {
+            font-family: Vazirmatn, sans-serif;
+          }
+        }
+
+        .page-head {
+          &,
+          h2,
+          h4 {
+            .page-title {
+              font-size: 1.625rem;
+            }
+          }
+        }
+
+        #dashboard {
+          #dashproducts {
+            nav {
+              font-family: Vazirmatn, sans-serif;
+            }
+          }
+
+          #dashtrends {
+            dt {
+              font-family: Vazirmatn, sans-serif;
+            }
+          }
+
+          section {
+            >section {
+              header {
+                .small {
+                  font-family: Vazirmatn, sans-serif;
+                }
+              }
+            }
+          }
+
+          .tooltip-panel-heading {
+            font-family: Vazirmatn, sans-serif;
+          }
+
+          svg {
+            text {
+              font-family: Vazirmatn, sans-serif;
+            }
+          }
+        }
+
+        .h1,
+        .h2,
+        .h3,
+        .h4,
+        .h5,
+        .h6,
+        h1,
+        h2,
+        h3,
+        h4,
+        h5,
+        h6,
+        input,
+        button,
+        textarea,
+        select {
+          font-family: Vazirmatn, sans-serif !important;
+        }
+
+        .nav-tabs {
+          li {
+            a {
+              font-family: Vazirmatn, sans-serif;
+            }
+          }
+        }
+      }
+    }
+
+    input {
+      &[type="email"],
+      &[type="password"],
+      &[type="search"],
+      &[type="tel"],
+      &[type="text"] {
+        font-family: Vazirmatn, sans-serif !important;
+      }
+    }
+
+    .ui-widget {
+      input,
+      select,
+      textarea,
+      button {
+        font-family: Vazirmatn, sans-serif !important;
+      }
+    }
+
+    .tooltip,
+    .ui-datepicker-rtl {
+      font-family: Vazirmatn, sans-serif;
+    }
+
+    #notification {
+      .dropdown-menu {
+        .notifications {
+          .nav-tabs {
+            .nav-item {
+              .nav-link {
+                font-family: Vazirmatn, sans-serif;
+              }
+            }
+          }
+        }
+      }
+    }
+
+    #login-panel {
+      input {
+        &[type="email"],
+        &[type="password"],
+        &[type="text"] {
+          font-family: Vazirmatn, FontAwesome, sans-serif !important;
+        }
+      }
+
+      #forgot_name {
+        font-family: Vazirmatn, sans-serif;
```

Comment:
All of these font-family props are here because we added some font family to LTR selectors?

Reply:
Yes!
And most of them used `!important` in CSS rules.